src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -656,7 +675,7 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
       )
     }
 
-    if (status === TxStatus.Failed) {
+    if (tx?.status === TxStatus.Failed) {
```

Comment:
Failed is the only case we leverage tx slice Tx for - we won't pick THOR upstream failures (i.e THORFi errors in this case) but we may well pick some at broadcast time
Realistically speaking, this was and still probably is superfluous, as this would really only be for broadcast errors, which is generally not the issue, though it doesn't *hurt* to keep for the sake of completeness!